packages/app/src/app/overmind/namespaces/comments/actions.ts

```diff
@@ -155,8 +155,8 @@ export const selectComment: AsyncAction<{
   } else {
     state.comments.currentCommentId = commentId;
     state.comments.currentCommentPositions = {
-      trigger: bounds,
-      dialog: null,
+      trigger: { right: window.innerWidth / 3, top: 0 },
+      dialog: { left: 0, top: 40 },
```

Comment:
```suggestion
      trigger: bounds,
      dialog: null
```

To avoid having duplicate defaults we can rather pass in the bounds from the outside (see below) and then depend on the defaults set by the component 😄